code/game/objects/items/rogueitems/keyrings.dm

```diff
@@ -227,11 +227,14 @@
 	keys = list(/obj/item/roguekey/warden, /obj/item/roguekey/walls)
 
 /obj/item/storage/keyring/guardcastle
```

Comment:
Then the keyring's not guardcastle anymore, is it.

Reply:
They can still access all parts of the castle, just not the manor inside of said castle.